app/extlinux_control.py

```diff
@@ -195,12 +238,12 @@ def get_standby_slot_dev(cls) -> str:
 
     @classmethod
     def get_standby_slot_partuuid(cls) -> str:
```

Comment:
This function returns APP or APP_b partition UUID instead of `get_standby_slot_dev`()'s partition.
APP or APP_b partition UUID should be used for extlinux.conf `PARTUUID=` specifier.